### DISCUSSION TITLE
'no decode delegate for this image format `CGM'

### DISCUSSION THREAD

**Discussion by Spooninfotech** [dlemstra/Magick.NET#722] [Help]
Hi,

I am getting this error when I am trying to covert "CGM" file.

ImageMagick.MagickMissingDelegateErrorException: 'no decode delegate for this image format `CGM' @ error/constitute.c/ReadImage/562'

**Comment by dlemstra** [comment]
It looks like you need to have this application http://www.agocg.ac.uk/train/cgm/ralcgm.htm in your `%PATH%`.
